core/registry/servicecenter/servicecenter.go

```diff
@@ -312,9 +312,10 @@ func (r *ServiceDiscovery) GetMicroServiceInstances(consumerID, providerID strin
 
 // FindMicroServiceInstances find micro-service instances
 func (r *ServiceDiscovery) FindMicroServiceInstances(consumerID, appID, microServiceName, version, env string) ([]*registry.MicroServiceInstance, error) {
```

Comment:
change inputs directly to FindMicroServiceInstances(consumerID, microServiceName,tags)